x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/mergers/rule_source/calculate_is_customized.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RuleResponse } from '../../../../../../../../common/api/detection_engine';
+import { MissingVersion } from '../../../../../../../../common/api/detection_engine';
+import type { PrebuiltRuleAsset } from '../../../../../prebuilt_rules';
+import { calculateRuleFieldsDiff } from '../../../../../prebuilt_rules/logic/diff/calculation/calculate_rule_fields_diff';
+import { convertRuleToDiffable } from '../../../../../prebuilt_rules/logic/diff/normalization/convert_rule_to_diffable';
+import { convertPrebuiltRuleAssetToRuleResponse } from '../../converters/convert_prebuilt_rule_asset_to_rule_response';
+
+export function calculateIsCustomized(
+  baseRule: PrebuiltRuleAsset | undefined,
+  nextRule: RuleResponse
+) {
+  if (baseRule == null) {
+    // If the base version is missing, we consider the rule to be not customized
```

Comment:
I gave some thought to this topic while writing the RFC and ended up coming to the opposite conclusion. My reasons are below, but open to hear what you think about it.

https://github.com/elastic/kibana/blob/fac236e567e60d21d92f1de907c97c7ae629e377/x-pack/plugins/security_solution/docs/rfcs/detection_response/prebuilt_rules_customization.md?plain=1#L1787-L1800

Reply:
Thanks for catching that, @jpdjere 👍 I fully agree that we should treat rules with missing `base` as customized.